enterprise/web.jsf/src/org/netbeans/modules/web/jsf/wizards/TemplateClientPanelVisual.java

```diff
@@ -75,7 +75,7 @@
 
     private WizardDescriptor wizardDescriptor;
 
-    private final Set/*<ChangeListener>*/ listeners = new HashSet(1);
```

Comment:
`HashSet` of size 1 is probably a non-sense. By default it has `75%` fill ratio and then it is rehashed.E.g. as soon as first item is added, the set is probably rehashed.

Reply:
> `HashSet` of size 1 is probably a non-sense. 

Just a follow-up comment in case someone comes across this bug report in the future.. I completely agree with your comment! My goal is simply to try and stay away from any "potential" source changes that could be behavioral. Even ones as small as this. Hope that explains why I do what I do..